.github/copilot-instructions.md

```
@@ -138,6 +138,193 @@ PolySim uses a centralized route manifest system for type-safe navigation and co
+### Misc
+
+- [Collapsible](https://ui.shadcn.com/docs/components/collapsible): Collapsible container component.
+- [Toggle](https://ui.shadcn.com/docs/components/toggle): Toggle button component.
+- [Toggle Group](https://ui.shadcn.com/docs/components/toggle-group): Group of toggle buttons.
+- [Pagination](https://ui.shadcn.com/docs/components/pagination): Pagination component for lists and tables.
+
+## Dark Mode
+
+- [Dark Mode](https://ui.shadcn.com/docs/dark-mode): Overview of dark mode implementation.
+- [Dark Mode - Next.js](https://ui.shadcn.com/docs/dark-mode/next): Dark mode setup for Next.js.
+# shadcn/ui
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)